lighthouse-core/test/audits/dobetterweb/doctype-test.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../../../audits/dobetterweb/doctype.js');
+const assert = require('assert');
+
+/* eslint-env mocha */
+
+describe('DOBETTERWEB: doctype audit', () => {
+  it('fails when HTML does not contain a doctype', () => {
+    const auditResult = Audit.audit({
+      Doctype: null,
+    });
+    assert.equal(auditResult.rawValue, false);
+  });
```

Comment:
also want to test the empty string, non-"html", and "html" success cases (as well as `publicId` and `systemId` if added?)